assets/lang/sklmessages_pl_PL.properties

```diff
@@ -80,8 +80,8 @@ editor.save=Zapisz profil
 
 settings.title=Ustawienia launchera
 settings.lang=J\u0119zyk
-settings.lang.help=Don't see your language?
-settings.lang.link=Help us translate!
+settings.lang.help=Widzisz błąd w tłumaczeniu?
+settings.lang.link=Pomóż go naprawić!
```

Comment:
"Don't see your language?" in Polish doesn't sound "Widzisz błąd w tłumaczeniu?" (and "Help us translate!" doesn't sound "Pomóż go naprawić!"). Is this a mistake, or maybe an announcement of future changes?

Reply:
I'll remove these strings anyway, in next update, these strings will only appear when English is selected.

Reply:
OK. Thanks for your reply.